gh-actions/release-images/action.yaml

```
@@ -14,15 +14,20 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - shell: bash
+    - name: Set up QEMU (to support building on non-native architectures)
+      uses: docker/setup-qemu-action@27d0a4f181a40b142cce983c5393082c365d1480
+    - name: Set up buildx
+      uses: docker/setup-buildx-action@94ab11c41e45d028884a99163086648e898eed25
+    - name: Build new images
+      # This needs to be kept separate so that the release stage runs using the new Shipyard base image
+      shell: bash
       env:
         IMAGES_ARGS: --nocache ${{ inputs.image_args }}
       run: |
         echo "::group::Build new images"
-        make images
+        make images multiarch-images
         echo "::endgroup::"
-
-    - name: Release newly built images
+    - name: Release newly build images
```

```suggestion
    - name: Release newly built images
```

Or I think there's a UK way like:

```suggestion
    - name: Release new-build images
```

---

Right, I meant “built”; “new-build images” doesn’t make sense even in the UK ;-).

---

https://dictionary.cambridge.org/us/dictionary/english/new-build
https://english.stackexchange.com/questions/270366/what-is-the-word-for-a-new-built-property